packages/website/docs/autocomplete-js.md

```diff
@@ -343,5 +343,27 @@ const {
   setStatus,
   setContext,
   refresh,
+  update,
+  destroy,
 } = autocomplete(options);
 ```
+
+## Helpers
```

Comment:
Since they're in "Returns", not sure about this new section?

Reply:
I figured it would be better to label them, because we don't document everything else that the `autocomplete` function returns. They're documented elsewhere (state, prop getters) so if we don't have a heading it feels like we forgot to document the others. wdyt?

Reply:
I would rather see something like this:

![image](https://user-images.githubusercontent.com/6137112/114696605-21b42a80-9d1d-11eb-8245-520a3a0438dc.png)